### ISSUE TITLE
TestRunner Logs view does not scroll beyond page (2.40.1)

### ISSUE THREAD

**Issue by testphreak** [anshooarora/extentreports#309]
Hey Anshoo! Nice work with the new features :smiley:. In my brief usage with the new version, the TestRunner Logs view did not scroll. Can you check if you're seeing this too?

**Comment by anshooarora** [comment]
I am able to reproduce.  You can apply this style for fix & I will include this in the next version by default:

```
#testrunner-logs-view > .col > .card-panel {
    bottom: 0;
    overflow-y: scroll;
    position: fixed;
    top: 50px;
    width: 95%;
}
```

**Comment by testphreak** [comment]
Thanks! Now scrolls with the custom style.

**Comment by anshooarora** [comment]
#321 - same issue with the Dashboard view, requires scrolling.

**Comment by anshooarora** [comment]
Fixed in 2.40.2-beta1.

**Comment by testphreak** [comment]
Thanks for the fix Anshoo!
